protocol/x/clob/memclob/memclob.go

```diff
@@ -964,10 +967,17 @@ func (m *MemClobPriceTimePriority) ReplayOperations(
 		// Matches are a no-op.
 		case *types.InternalOperation_Match:
 		case *types.InternalOperation_OrderRemoval:
+			orderId := operation.GetOrderRemoval().OrderId
```

Comment:
Question: do you think it's worth keeping track of which order IDs have already been removed, and then error logging if you see multiple order removals? I believe we currently swallow that error if it occurs.